src/utils.js

```diff
@@ -102,6 +102,16 @@ const removeAlreadyChallengedPlayers = (players, playerId, in_progress) => {
 
 const removeEmptyArray = array => array.filter(i => (i.length === 0) ? false : true)
 
+const eloCalculation = (playerAElo, playerBElo, playerAScore, playerBScore) => {
+    const probabilityOfWinPlayerA = 1.0 * 1.0 / (1 + 1.0 * Math.pow(1, 1.0 * (playerAElo - playerBElo) / 400))
+    const probabilityOfWinPlayerB = 1.0 * 1.0 / (1 + 1.0 * Math.pow(1, 1.0 * (playerBElo - playerAElo) / 400))
+
+    return {
+        playerANewElo: playerAElo + 10 * (playerAScore - probabilityOfWinPlayerA),
```

Comment:
You're fixing the *K* factor to 10. The recommendation is to change that based on how strong/weak the players are. __(see image below)__
![image](https://user-images.githubusercontent.com/7025727/65915407-054dd880-e399-11e9-8772-dee8351c9c67.png)

Reply:
Agree.

Reply:
Do you think we should use the USCF values? `32` for players under 2100, `24` for players between 2100 and 2400 and `16` for player over 2400?